ngraph/core/include/ngraph/function.hpp

```diff
@@ -150,6 +169,7 @@ namespace ngraph
         topological_sort_t m_topological_sorter;
 
         ResultVector m_results;
+        SinkVector m_sinks;
```

Comment:
Please provide description in a comment for m_sinks: how they are different from m_results.